src/autoscaler/pruner/cmd/pruner/pruner_test.go

```diff
@@ -73,9 +73,19 @@ var _ = Describe("Pruner", func() {
 				}))
 		})
 
-		It("should start", func() {
```

Comment:
if you separate into multiple test cases, then you don't need duplicate test of 
Eventually(runner.Session.Buffer, 2*time.Second).Should(Say("pruner.started")) Consistently(runner.Session).ShouldNot(Exit())

You can have another test case to test for "prunder started"